### PR TITLE
Initial lib

### PR DESCRIPTION
**Pull request by immortal-tofu**

### CODE REVIEW

File: Cauliflower.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      });
+    }
+    this.patternCallbacks.filter((handler) => {
+      if (handler.id === id) return false;
+      return true;
+    });
+  }
+
+  throw (identifier, ...args) {
+    console.log(this.simpleCallbacks);
```
Comment (adlenafane) [inline Cauliflower.js]: Console

File: Cauliflower.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    if (this.simpleCallbacks[identifier]) {
+      this.simpleCallbacks[identifier].forEach((handler) => {
+        console.log(handler);
```
Comment (adlenafane) [inline Cauliflower.js]: Console

**Comment by adlenafane** [comment]
Overall LGTM. 
Some doc on how to use would be welcome. 
And a couple of tests would also be great to start on good basis

**Comment by immortal-tofu** [comment]
Yes I'll do this :)

**Comment by adlenafane** [comment]
LGTM :rocket:

**Pull request merged**